scheduler.go

```diff
@@ -207,10 +232,26 @@ func (sc *Scheduler) runService(ctx context.Context, task Task) error {
 	select {
 	case res := <-ok:
 		if res.Error != nil {
-			return fmt.Errorf("service %s: %s", task.Service, res.Error.Message)
+			return fmt.Errorf("message: %s", res.Error.Message)
 		}
 		if res.StatusCode != 0 {
-			return fmt.Errorf("service %s: status code %d", task.Service, res.StatusCode)
+			reader, err := sc.client.ContainerLogs(ctx, task.Container, types.ContainerLogsOptions{
```

Comment:
I think seeing the container logs in the error message can be more useful than just the status code